test/extensions/formatter/cel/BUILD

```diff
@@ -0,0 +1,28 @@
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_package",
+)
+load(
+    "//test/extensions:extensions_build_system.bzl",
+    "envoy_extension_cc_test",
+)
+
+licenses(["notice"])  # Apache 2
+
+envoy_package()
+
+envoy_extension_cc_test(
+    name = "cel_test",
+    srcs = ["cel_test.cc"],
```

Comment:
Can you use copts here for windows and then macro USE_CEL_PARSER?
Windows problem is temporary - it's caused by ANTLR failing to build, nothing inherent.